iree/turbine/kernel/wave/wave.py

```diff
@@ -336,7 +336,8 @@ def test_execute(self, args, kwargs):
 
         run = kwargs.get("run", False)
         run_bench = kwargs.get("run_bench", False)
-        if run or run_bench:
+        dry_run = kwargs.get("dry_run", False)
```

Comment:
Do we even need this flag? If you set run = False and run_bench = False and set dump_vmfb_file, doesn't that work?

Reply:
If we set both `run` and `run_bench` to `False` we will never enter this path. Although, as you mentioned it, I think instead of adding `dry_run` flag I can instead promote `dump_vmfb` flag from run config to general flag. Let me try it.

Reply:
done

Reply:
thanks